scrape/scrape.go

```diff
@@ -731,7 +744,7 @@ func mutateReportSampleLabels(lset labels.Labels, target *Target) labels.Labels
 }
 
 // appender returns an appender for ingested samples from the target.
-func appender(app storage.Appender, limit int) storage.Appender {
+func appender(app storage.Appender, limit, bucketLimit int) storage.Appender {
```

Comment:
limit → sampleLimit (because it's confusing now that we have two different limits).